webapp/src/main/webapp/templates/freemarker/page/partials/developerPanel.ftl

```diff
@@ -19,6 +19,9 @@
 <#elseif !settings.mayControl>
 	<div class="developer">
 		<h1>${siteName} is running in developer mode.</h1>
+		<div id="publicSettings" style="display:none;">
+			<@showCheckbox "developer_i18n_onlineTranslation","Enable online translation" />
+		</div>
```

Comment:
Apologies, I didn't get the purpose of this code. Can you please check if this code is necessary?

Reply:
It is an option in developer's panel to turn on/off online translations

Reply:
And what is the purpose of lines 62 and 63 of this file?

Reply:
Oh. Sorry I got mixed up. I forgot about it as it was a long time ago. This code is here to make online translations work for unauthorized users. When user is not authorized there is no developer panel on page, so part of the online translation code that runs in browser won't know if it's on or off.  And the 62-63 lines is to turn it on and off when the dev panel is on page.

Reply:
Thanks for clarification.